allofplos/corpus_helpers.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+import hashlib
+
+import requests
+
+from .transformations import BASE_URL_API
+
+BLOCKSIZE = 65536
+hasher = hashlib.sha256()
+
+
+def hash_file(fname):
+    """ Create a SHA-256 hash for an article file in the corpus directory.
+
+    Used in `Corpus().hashtable()`. Takes a full filepath.
+    :return: SHA-256 hexcode for a file
+    """
+    with open(fname, 'rb') as afile:
+        buf = afile.read(BLOCKSIZE)
+        while len(buf) > 0:
+            hasher.update(buf)
+            buf = afile.read(BLOCKSIZE)
+        file_hash = hasher.hexdigest()
+
+    return file_hash
+
+
+def search_solr_records(days_ago=14, start=0, rows=1000, start_date=None, end_date=None, item='id'):
```

Comment:
This isn't specific to the corpus but seems to be an interactive api (not a library api); it probably should exist not only in a separate PR but possibly a separate module.